src/network/netdev/netdev-gperf.gperf

```diff
@@ -40,7 +40,7 @@ Match.Architecture,                config_parse_net_condition,           CONDITI
 NetDev.Description,                config_parse_string,                  0,                             offsetof(NetDev, description)
 NetDev.Name,                       config_parse_ifname,                  0,                             offsetof(NetDev, ifname)
 NetDev.Kind,                       config_parse_netdev_kind,             0,                             offsetof(NetDev, kind)
-NetDev.MTUBytes,                   config_parse_iec_size,                0,                             offsetof(NetDev, mtu)
+NetDev.MTUBytes,                   config_parse_mtu,                     0,                             offsetof(NetDev, mtu)
```

Comment:
I think `AF_UNSPEC` should be used here instead of `0` for consistency to other places.

Reply:
true!